<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditEmployeeForm</class>
 <widget class="QDialog" name="EditEmployeeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1166</width>
    <height>586</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1166</width>
    <height>586</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1166</width>
    <height>586</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit/Delete Employee</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#EditEmployeeForm{
	background-color:  #C2272D;
}</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>731</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="updateDetailsButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Update Details</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteEmployeeButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>430</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete User/Employee</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addEmployeeButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Employee</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshTableButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>530</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh Table</string>
   </property>
  </widget>
  <widget class="QLabel" name="instructionLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>520</y>
     <width>601</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Double click on a table record's &amp;quot;userid&amp;quot; value  to load them in the input fields on the left. Click the &amp;quot;Refresh Table&amp;quot; button&lt;/p&gt;&lt;p&gt;to refresh this table. To edit a user's details, edit the input fields and click &amp;quot;Update Details&amp;quot;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>30</y>
     <width>361</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_1">
        <property name="text">
         <string>User id</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="userid">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>User level id</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="userlevelid">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>User name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="username">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>First name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="firstname">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Last name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lastname">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="email">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Age</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="age">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Status</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="status">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
